server/game/gamesteps/actionwindow.js

```diff
@@ -22,9 +22,9 @@ class ActionWindow extends PlayerOrderPrompt {
 
     activePrompt() {
         return {
-            menuTitle: 'Any actions or reactions?',
+            menuTitle: 'Initiate an action, or pass',
             buttons: [
-                { text: 'Done' }
+                { text: 'Pass' }
```

Comment:
This made a whole lot of integration tests explode, as they're expecting to find "Done" in the dialog window.
I guess you were waiting for feedback on the proposal before fixing those, but just wanted to note down here that, if accepted, they'll need to be fixed before this can go in.